### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by mbdesigns1989**
Portfolio: desktop version
Add markup to the Homepage and style (mobile-first then media query).
No Linters errors.

### CODE REVIEW

**Comment by mbdesigns1989** [comment]
Please, I'm repeating the week so every review 
count, I don't want to repeat the module, thanks.

File: index.html [hunk 1]
```diff
@@ -1,241 +1,310 @@
...
-<header>
-  <nav class="d-flex header">
-    <a href="#" class="logo  nav-link"><span>My Logo</span></a>
-    <span class="material-icons nav-link" >menu</span>
-  </nav>
-</header>
+        <section class="second-section-container">
+          <section class="second-section">
+            <p>I'm Mohamed Badr</p>
+            <p>Glad to see you!</p>
```
Comment (mimipeshy) [inline index.html]: - [ ] I think it would be best if this was an `h1` tag instead of a `p`   tag since its the main title of the page

**Review by mimipeshy — CHANGES_REQUESTED** [review]
## Hi @mbdesigns1989  👋🏾 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
#### Highlights

- Your desktop version looks  nice ✔️ 
- Good GitHub workflow ✔️ 
- Your mobile view has not been distorted ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](<https://microverse.zendesk.com/hc/en-us/articles/1500004088561>) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](<https://airtable.com/shrQAqnBwek5a0O0s>)._

**Review by hemantso — APPROVED** [review]
Hi @mbdesigns1989 and @Kossi-stack , 

Well done on making the requested changes. 🚀 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

## Congratulations! 🎉 

Alt Text

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- To make our live link more presentable in the `README` file, we can insert the link like this `[live demo](https://raw.githack.com/mbdesigns1989/my-portfolio-website/desktop-version/index.html)`. 

Cheers,
Hemant Soni
Cheers and happy coding 😃

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. You can mention me (@hemant-soni-vst-au4) so I can see your comments and respond.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**